node/migrations/node/000030_remove_wallets.down.sql

```diff
@@ -0,0 +1,4 @@
+CREATE TABLE IF NOT EXISTS wallets (
+    id SERIAL PRIMARY KEY,
+    pk TEXT NOT NULL UNIQUE
+)
```

Comment:
looks like `up` and `down` file contents got swapped? `down` is creating and `up` is dropping

Reply:
yes, I'm dropping wallets table, it should be `up`  😊